python/cuml/linear_model/logistic_regression.pyx

```diff
@@ -198,8 +201,14 @@ class LogisticRegression(Base):
 
         self.qn = QN(loss=loss, fit_intercept=self.fit_intercept,
                      l1_strength=l1_strength, l2_strength=l2_strength,
-                     max_iter=self.max_iter, tol=self.tol,
-                     verbose=self.verbose, handle=self.handle)
+                     max_iter=self.max_iter,
+                     linesearch_max_iter=self.linesearch_max_iter,
+                     tol=self.tol, verbose=self.verbose, handle=self.handle)
+
+        if self.verbose > 1:
+            self.verb_prefix = "CY::"
```

Comment:
My reasoning is that if a detailed printing is desired, knowing exactly where messages are originated (cython vs c++ vs dask etc) then we want to prefix where it is happening. We could be more detailed with something like `CY/LogisticRegression::` so that'll help debugging and optimizing estimators. So my idea would be to have a standard like `CY` for originating in cython, `DA` for originating in Dask related code, `CP` for pure C++ and `CU` for cuda or something like that. Needs some thought, but I think it might be pretty valuable, what do you think?

Reply:
Oh, it's for cython... I would just make it `Cython:`
I think everyone will see this and wonder what Cy means